docs/source/quickstart/02_prepare_env.md

```diff
@@ -75,30 +76,35 @@ hypervisors:
 ~~~
 
 As you can see, two custom parameters are passed via the inventory: `vxlan_remote_ip` and `vxlan_local_ip`.
-Those are needed only if you're provisioning a VXLAN connection between your two hypervisors using
-[this provided playbook](../files/bootstrap-vxlan.yml).
+Those are needed only if you're provisioning a VXLAN connection between your two hypervisors using `docs/source/files/bootstrap-vxlan.yml`.
 
 ## Bootstrap hypervisor
+
 Since we're using non-root user with some specificities, you may want to get an automated way to provision the
 hypervisor.
 
 ### Basics
-The [following playbook](../files/boostrap-hypervisor.yml) will help you create the user, ensuring some
+
+The boostrap-hypervisor.yml will help you create the user, ensuring some
 packages are present, as well as ensuring your user will be part of the needed group.
 
 You can run the playbook like this:
+
 ```Bash
+$ cd ci-framework
 $ ansible-playbook -i custom/inventory.yml \
-    -U root \
-    -e cifmw_target_host=[hypervisor|hypervisors]
-    bootstrap-hypervisor.yml
+    -e ansible_user=root \
+    -e cifmw_target_host=[hypervisor|hypervisors] \
+    docs/source/files/bootstrap-hypervisor.yml
 ```
 
 ### VXLAN
-Once you're finished with the basic bootstrap, and if you're having two hypervisors, you can consume the
-[bootstrap-vxlan.yml](../files/bootstrap-vxlan.yml):
+
+Once you're finished with the basic bootstrap, and if you're having two hypervisors, you can consume the `bootstrap-vxlan.yml`:
```

Comment:
After completing the initial bootstrap process, if you are deploying on two hypervisors, you can consume the 'bootstrap-vxlan.yml' playbook

Reply:
@rebtoor care to address this? @lewisdenny's proposal sounds better.

Reply:
Sure! Thanks @lewisdenny !